util/downloadUrlByOS.ts

```diff
@@ -0,0 +1,48 @@
+import type { UserOS } from '../types/userOS';
+
+const X64_BITNESS = '64';
+
+export const downloadUrlByOS = (
```

Comment:
The arguments can be an object instead of 4 different params.

Reply:
I'm trying to get a feel of the code guideline, is there a markdown to read?
I was hoping to have additional feedback from the linter.